### PR TITLE
add gem icon

### PR DESCRIPTION
**Pull request by connium**
Co-authored-by: b0b90 <babersheikh@outlook.com>

gem

### CODE REVIEW

**Comment by connium** [comment]
I've taken the gem icon from @b0b90 which he provided in #350. I have optimized the paths and formatted the file svg file to match the [Icon Design Guide](https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md).

**Comment by connium** [comment]
Is there anything that I should change to get this merged?

**Comment by locness3** [comment]
Doesn't fit much with the guidelines to me.

**Comment by connium** [comment]
Can you please explain in more detail what doesn't fit the guidelines from your point of view?

**Comment by locness3** [comment]
Feels too dense.

**Comment by JanTrichter** [comment]
I think the edges are too sharp. This is one of those icons that are basically impossible to create with the padding from the style guide.

Also tried to create my own version but I am not really convinced:
https://diamond-comparison-cmx9st4f5-jantrichter.vercel.app/

**Comment by ericfennis** [comment]
@JanTrichter I this case the padding is not deeded, because it is one element. I agree with @locness3 that the lines looks a bit too dense to each other.

**Comment by ericfennis** [comment]
Less dens alternative:
image

**Comment by connium** [comment]
@ericfennis I like your proposal.

File: icons/gem.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <polygon points="6 2 18 2 22 9 12 22 2 9" />
+  <line x1="2" y1="9" x2="22" y2="9" />
+  <polyline points="6 2 7 9 12 22 17 9 18 2" />
+  <polyline points="7 9 12 2 17 9" />
```
Comment (ericfennis) [inline icons/gem.svg:15]: Here is the svg code
```suggestion
  <path d="m22 9 .8.6a1 1 0 0 0 0-1.2L22 9ZM12 22l-.8.6a1 1 0 0 0 1.6 0L12 22ZM2 9l-.8-.6a1 1 0 0 0 0 1.2L2 9Zm4-6V2a1 1 0 0 0-.8.4L6 3Zm12 0 .8-.6A1 1 0 0 0 18 2v1Zm3.2 5.4-10 13 1.6 1.2 10-13-1.6-1.2Zm-8.4 13-10-13-1.6 1.2 10 13 1.6-1.2ZM6 4h12V2H6v2Zm-.8-1.6-4 6 1.6 1.2 4-6-1.6-1.2Zm17.6 6-4-6-1.6 1.2 4 6 1.6-1.2Z" />
  <path d="m12 22 4-13-3-6" />
  <path d="M12 22 8 9l3-6" />
  <path d="M2 9h20" />
```

Comment (ericfennis) [inline icons/gem.svg:15]: @connium Here is the code.
```suggestion
  <polygon points="6 3 18 3 22 9 12 22 2 9" />
  <path d="m12 22 4-13-3-6" />
  <path d="M12 22 8 9l3-6" />
  <path d="M2 9h20" />
```

**Comment by connium** [comment]
Thanks for your support!

**Review by ericfennis — APPROVED** [review]

**Comment by ericfennis** [comment]
@connium No problem!

**Pull request merged**